### ISSUE TITLE
First line of invalid configuration logged, possibly leaking credentials

### ISSUE THREAD

**Issue by RichardoC** [aws/aws-cli#6696]
Confirm by changing [ ] to [x] below to ensure that it's a bug:
- [ x] I've gone though the [User Guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) and the [API reference](https://docs.aws.amazon.com/cli/latest/reference/)
- [x ] I've searched for [previous similar issues](https://github.com/aws/aws-cli/issues) and didn't find any solution
  

**Describe the bug**

**SDK version number**
aws --version
aws-cli/1.18.149 Python/3.9.6 Linux/5.4.0-96-generic botocore/1.18.8

**Platform/OS/Hardware/Device**
Ubuntu 20.04

**To Reproduce (observed behavior)**
``` bash
export AWS_SHARED_CREDENTIALS_FILE=/tmp/invalidcredfile
# where /tmp/invalidcredfile contains `mysecretfilethatshouldn'tbelogged` for below
# run any aws cli command, example
aws eks get-token --cluster-name example --profile example-profile --region eu-west2
```

**Expected behavior**
Only the filename and line number of the error logged. Not the contents of the file.

**Logs/output**

```python3
Traceback (most recent call last):
--
File "redacted/aws.pex/third_party/python3/botocore/configloader.py", line 149, in raw_config_parse
File "redacted/python/3.9.6/usr/lib/python3.9/configparser.py", line 697, in read
self._read(fp, filename)
File "redacted/python/3.9.6/usr/lib/python3.9/configparser.py", line 1085, in _read
raise MissingSectionHeaderError(fpname, lineno, line)
 configparser.MissingSectionHeaderError: File contains no section headers.
file: '/tmp/invalidcredfile', line: 1

Traceback (most recent call last):
  File "redacted/botocore/configloader.py", line 1[49]in raw_config_parse
  File "redacted/python/3.9.6/usr/lib/python3.9/configparser.py", line 697, in read
    self._read(fp, filename)
  File "redacted/python/3.9.6/usr/lib/python3.9/configparser.py", line 1085, in _read
    raise MissingSectionHeaderError(fpname, lineno, line)
configparser.MissingSectionHeaderError: File contains no section headers.
file: '/tmp/invalidcredfile', line: 1
'mysecretfilethatshouldn'tbelogged'
```

**Additional context**

**Comment by nateprewitt** [comment]
We've deployed a fix for this in boto/botocore#2607. This should be available in today's release.

**Comment by tim-finnigan** [comment]
Thanks @RichardoC for bringing this to our attention and @nateprewitt for the quick fix! I’ll go ahead and close the issue.
